javascript/src/URDFClasses.js

```diff
@@ -228,7 +238,23 @@ class URDFJoint extends URDFBase {
 
         }
 
-        return false;
+        return didUpdate;
+
+    }
+
+}
+
+class URDFMimicJoint extends URDFJoint {
+
+    constructor(...args) {
+        super(...args);
+        this.offset = 0;
+        this.multiplier = 1;
+    }
```

Comment:
Lets initialize `mimicJoint` to `null` in the constructor since we set it in the loader.

Reply:
Woops! Done.